lib/src/components/chart_style.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:ui';
+
+class ChartStyle {
+  final Color? barColor;
+  final Color? tooltipBackgroundColor;
+  final Color? labelColor;
+  final Color? verticalGridColor;
+  final Color? horizontalGridColor;
+
+  ChartStyle({
```

Comment:
I think we can make this as const.
```suggestion
  const ChartStyle({
```